pytest_splunk_addon/standard_lib/addon_parser/__init__.py

```diff
@@ -49,25 +45,15 @@ class AddonParser(object):
 
     def __init__(self, splunk_app_path):
         self.splunk_app_path = splunk_app_path
-        LOGGER.info(
```

Comment:
What about decorator we could add to logger lib
Could be used for functions to log info level entry and exit points - function name & argument list and exit values respectively

Reply:
I was using this kind of functionality years ago in C#
It was extremely useful

Reply:
Great improvement for the future :)